added home page content and profile pictures

ashwoodaa opened the pull request:
it is telling me that I can't automatically merge

ashwoodaa committed:
added home page content and profile pictures

ashwoodaa commented:
I made sure to pull before I did this to make sure I had everything that was up. I checked my status and it said everything was good. I can't figure out the conflict. Will look into it tomorrow.

josephkane commented on index.html:
```diff
@@ -26,26 +26,33 @@
-        <p></p>
+    <div id="columnCenter">
+      <div id="Joe">
+        <div class="profilePics"><img src="joeProfilePic.jpg" alt="Joe"></div></div><!--feel free to change-->
```
I think the image source url needs to be "img/joeProfilePic.jpg" etc

ashwoodaa replied:
cool. taken care of :+1:

ashwoodaa committed:
fixed file paths for Profile pics                       still working on
everything else

ashwoodaa committed:
fixed picture file paths on testimonials page

MeganWord commented:
:+1: nice work!

ashwoodaa committed:
removed some random unneed header tags at the bottom and also closed a lot
of div tags that needed closing and commented on the closed tags so it is
super organized
